app/src/main/java/edu/uc/groupProject/topten/ui/main/CurrentListAdapter.kt

```diff
@@ -0,0 +1,76 @@
+package edu.uc.groupProject.topten.ui.main
```

Comment:
Moved the CurrentListAdapter class from DAO to ui.main package.
As I see it to be a front-end, or UI focused, class that deals with already existing data available from MainViewModel, and deals with programmatically adding a list to a Recycler view container using an adapter.

Reply:
Yes... Adapter, as a UI class, should be in UI package, not DAO.